src/Plugin/Derivative/TranslationLocalTasks.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\oe_translation\Plugin\Derivative;
+
+use Drupal\Component\Plugin\Derivative\DeriverBase;
+use Drupal\Core\Plugin\Discovery\ContainerDeriverInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\oe_translation\TranslatorProvidersInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Generates the local tasks for our translation system.
+ */
+class TranslationLocalTasks extends DeriverBase implements ContainerDeriverInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * The base plugin ID.
+   *
+   * @var string
+   */
+  protected $basePluginId;
+
+  /**
+   * The translation providers service.
+   *
+   * @var \Drupal\oe_translation\TranslatorProvidersInterface
+   */
+  protected $translatorProviders;
+
+  /**
+   * Creates a FieldUiLocalTask object.
+   *
+   * @param string $base_plugin_id
+   *   The base plugin ID.
+   * @param \Drupal\oe_translation\TranslatorProvidersInterface $translatorProviders
+   *   The translation providers service.
+   */
+  public function __construct($base_plugin_id, TranslatorProvidersInterface $translatorProviders) {
+    $this->basePluginId = $base_plugin_id;
+    $this->translatorProviders = $translatorProviders;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, $base_plugin_id) {
+    return new static(
+      $base_plugin_id,
+      $container->get('oe_translation.translator_providers'),
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDerivativeDefinitions($base_plugin_definition) {
```

Comment:
```suggestion
  public function getDerivativeDefinitions($base_plugin_definition): array {
```

Reply:
Parent doesn't have it, not our interface.